tools/eslint-rules/no-unescaped-regexp-dot.js

```diff
@@ -16,10 +16,29 @@ module.exports = function(context) {
 
   function report(node, startOffset) {
     const indexOfDot = sourceCode.getIndexFromLoc(node.loc.start) + startOffset;
+    const expression = sourceCode.getText(node);
+    const dotOffset = expression.indexOf('.');
+    var correctedExpression = '';
+
+    if (expression.charAt(dotOffset - 1) === '\\') {
```

Comment:
I don't think this code should run unless a fixer is requested.

Reply:
@apapirovski Fixed.